frontend/src/Dashboard/Header/Header.js

```diff
@@ -24,8 +24,11 @@ class Header extends React.Component {
         <div className="Header__main">
           <h1 className="Header__h1">{"CTXfloods Dashboard"}</h1>
           <ul className="Header__tabs">
-            <li className={pathname.endsWith('crossings') ? 'Header__tab--active' : 'Header__tab'}>
-              <Link to="/dashboard/crossings">Crossing Updates</Link>
+            <li className={pathname.endsWith('crossings/list') ? 'Header__tab--active' : 'Header__tab'}>
+              <Link to="/dashboard/crossings/list">Crossing List</Link>
+            </li>
+            <li className={pathname.endsWith('crossings/map') ? 'Header__tab--active' : 'Header__tab'}>
```

Comment:
I'm probably wrong, but this logic feels a little brittle. If we change the pathnames, how would this fail?

Reply:
hmm, what would you suggest as a better solution?

Reply:
Yeah, I can't think of a better solution myself. Perhaps an option would be to pull out the pathnames and titles as key/values in an Array of objects that can be edited in one place. And then we `.map()` over them to create the `<link>` elements with its nested `<Link>`.

Meh, maybe over-engineering this non-issue?